docker/develop/Dockerfile

```diff
@@ -142,10 +142,10 @@ RUN git clone https://github.com/01org/tbb /tmp/tbb-2017_U7; \
     rm -rf /tmp/tbb-2017_U7
 
 # install docker
-RUN curl -L -o /tmp/docker-17.06.0-ce.tgz https://download.docker.com/linux/static/stable/x86_64/docker-17.06.0-ce.tgz; \
-    tar -xz -C /tmp -f /tmp/docker-17.06.0-ce.tgz; \
+RUN curl -L -o /tmp/docker-17.06.2-ce.tgz https://download.docker.com/linux/static/stable/x86_64/docker-17.06.2-ce.tgz; \
```

Comment:
Is there something special in the version 17.06.02-ce that does not exist in 17.06.0-ce?